app/models/group.rb

```diff
@@ -155,21 +155,13 @@ def delete_permission(current_user, removed_user)
   end
 
   def communities
-    values = []
     if code == "PPPL"
-      values << "Princeton Plasma Physics Laboratory"
+      ["Princeton Plasma Physics Laboratory"]
     else
-      values << "Princeton Neuroscience Institute"
-      values << "Department of Geosciences"
-      values << "Mechanical and Aerospace Engineering"
-      values << "Astrophysical Sciences"
-      values << "Civil and Environmental Engineering"
-      values << "Chemical and Biological Engineering"
-      values << "Digital Humanities"
-      values << "Music and Arts"
-      values << "Princeton School of Public and International Affairs"
+      ["Princeton Neuroscience Institute", "Department of Geosciences", "Mechanical and Aerospace Engineering",
```

Comment:
This was just to make rubocop happy about the class length.  I believe it is still readable.